src/types/jobDescription.ts

```diff
@@ -0,0 +1,356 @@
+                <vscode-data-grid-cell grid-column="2">${this.jobDescriptionInfo?.messageLoggingLevel}</vscode-data-grid-cell>
+              </vscode-data-grid-row>
+              <vscode-data-grid-row>
+                <vscode-data-grid-cell grid-column="1">Message Logging Severity</vscode-data-grid-cell>
+                <vscode-data-grid-cell grid-column="2">${this.jobDescriptionInfo?.messageLoggingSeverity}</vscode-data-grid-cell>
+              </vscode-data-grid-row>
+              <vscode-data-grid-row>
+                <vscode-data-grid-cell grid-column="1">Message Logging Text</vscode-data-grid-cell>
+                <vscode-data-grid-cell grid-column="2">${this.jobDescriptionInfo?.messageLoggingText}</vscode-data-grid-cell>
+              </vscode-data-grid-row>
+              <vscode-data-grid-row>
+                <vscode-data-grid-cell grid-column="1">Log CL Program Commands</vscode-data-grid-cell>
+                <vscode-data-grid-cell grid-column="2">${this.jobDescriptionInfo?.logClProgramCommands}</vscode-data-grid-cell>
+              </vscode-data-grid-row>
+              <vscode-data-grid-row>
+                <vscode-data-grid-cell grid-column="1">Device Recovery Action</vscode-data-grid-cell>
+                <vscode-data-grid-cell grid-column="2">${this.jobDescriptionInfo?.deviceRecoveryAction}</vscode-data-grid-cell>
+              </vscode-data-grid-row>
+              <vscode-data-grid-row>
+                <vscode-data-grid-cell grid-column="1">Time Slice End Pool</vscode-data-grid-cell>
+                <vscode-data-grid-cell grid-column="2">${this.jobDescriptionInfo?.timeSliceEndPool}</vscode-data-grid-cell>
+              </vscode-data-grid-row>
+              <vscode-data-grid-row>
+                <vscode-data-grid-cell grid-column="1">Allow Multiple Threads</vscode-data-grid-cell>
+                <vscode-data-grid-cell grid-column="2">${this.jobDescriptionInfo?.allowMultipleThreads}</vscode-data-grid-cell>
+              </vscode-data-grid-row>
+              <vscode-data-grid-row>
+                <vscode-data-grid-cell grid-column="1">Workload Group</vscode-data-grid-cell>
+                <vscode-data-grid-cell grid-column="2">${this.jobDescriptionInfo?.workloadGroup}</vscode-data-grid-cell>
+              </vscode-data-grid-row>
+              <vscode-data-grid-row>
+                <vscode-data-grid-cell grid-column="1">ASPGRP</vscode-data-grid-cell>
+                <vscode-data-grid-cell grid-column="2">${this.jobDescriptionInfo?.aspgrp}</vscode-data-grid-cell>
+              </vscode-data-grid-row>
+              <vscode-data-grid-row>
+                <vscode-data-grid-cell grid-column="1">DDM Conversation</vscode-data-grid-cell>
+                <vscode-data-grid-cell grid-column="2">${this.jobDescriptionInfo?.ddmConversation}</vscode-data-grid-cell>
+              </vscode-data-grid-row>
+            </vscode-data-grid>
+            `;
```

Comment:
It could be interesting to use `Components.dataGrid` here as well. That would require a few changes but it could reduce the UI code and ensure UI consistency:

- `jobDescriptionInfo` would be declared using the `Record<string, string>` type (to hold each title/value entries)
- each entry would be loaded from the fetched row
- `Components.dataGrid` would be used to display the resulting table

```
private jobDescriptionInfo : Record<string, string> = {};
...
jobDescriptionInfo["Job Description Library"] = String(row.jobDescriptionLibrary);
jobDescriptionInfo["Job Description"] = String(row.jobDescription);
...        
Components.dataGrid<[string, string]>({
    columns:[
        {title: "", cellValue: entry => entry[0]},
        {title: "", cellValue: entry => entry[1]}
    ]
}, Object.entries(jobDescriptionInfo));
```